### DISCUSSION TITLE
Need help and Discord is closed

### DISCUSSION THREAD

**Discussion by ScArWlvrne** [Universal-Team/Universal-Updater#75] [General]
I'm trying to get a unistore set up for the Ocarina of Time 3D Randomizer hack that released just yesterday. I don't really know what I'm doing, and I have one that I can't get to work, but the Discord server is currently closed to new members, so this was my next best option. If I could either be let into the Discord (ScAr_wlvrne#9296) or get some help, that would be great. I'll attach my unistore as a txt (it's  a WiP that I just used the example as a template for) if looking at that helps anybody.

[OoT3DR.txt](https://github.com/Universal-Team/Universal-Updater/files/6176715/OoT3DR.txt)

**Comment by Epicpkmn11** [comment]
You've got a comma on line 41 that's breaking it, JSON (which is what UniStores are) is very picky about formatting.

You can use https://jsonformatter.curiousconcept.com/ to check for errors (and fix some automatically if the "Fix JSON" box is ticked)
